python/py-h5py/Portfile

```diff
@@ -34,7 +34,7 @@ long_description  \
 
 homepage                https://www.h5py.org
 
-python.versions         36 37 38
+python.versions         36 37 38 39
 python.default_version  37
```

Comment:
```suggestion
```

Can we remove the `default_version 37`, since python37 was the latest version when this was added but shouldn't be necessary. Pinging @eborisch.

Reply:
@eborisch, any thoughts on this?

Reply:
Makes sense to remove, given that the vast majority of our py-* ports don't have it. Done.